ansible/library/exabgp.py

```diff
@@ -48,6 +49,7 @@
 http_api_py = '''\
 from flask import Flask, request
 import sys
+import six
```

Comment:
Duplicated `import six` here.

Reply:
This import here is part of `http_api_py` which is the HTTP API processor generated as `/usr/share/exabgp/http_api.py` on the PTF container and is not for the exabgp.py.